src/WaCore.Data.Ef/WacDbContextTransactionWrapper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.EntityFrameworkCore.Storage;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using WaCore.Contracts.Data;
+
+namespace WaCore.Data.Ef
+{
+    public class WacDbContextTransactionWrapper : IWacTransaction
```

Comment:
My personal opinion is that we can just name this class - WacDbContextTransaction